doc/release-notes-29091-29165.md

```diff
@@ -0,0 +1,5 @@
+Build
+-----
+
+GCC 11.1 or later, or Clang 15+ or later,
```

Comment:
nit: In the phrase "Clang 15+ or later" the plus sign seems redundant.

Reply:
Yea. Will merge this now. If clang-16 happens, it can be updated, if not, it can be changed in the wiki.